test/cypress/integration/compliance/small-business-lending/filing-instruction-guide.cy.js

```diff
@@ -1,223 +1,214 @@
 import { FilingInstructionGuide } from './filing-instruction-guide-helpers.cy.js';
-import { skipOn } from '@cypress/skip-test';
+import { onlyOn } from '@cypress/skip-test';
 
 const fig = new FilingInstructionGuide();
 
-/* TODO: Remove the before hook, it( 'skips tests if page does not exist')
-   test, and skipOn once the FIG page has been published */
-before( () => {
-  cy.request( {
-    url: fig.url(),
-    failOnStatusCode: false
-  } ).as( 'figPage' );
-} );
+describe( '1071 Filing Instruction Guide (FIG)', () => {
+
+  describe( 'FIG table of contents', () => {
+
+    /* Our FIG sample page only exists in certain environments so continue
+       this test suite only if the user explicitly provides a URL via
+       CYPRESS_FIG_URL=https://blah.cfpb.gov/xxxxxx/yyyyyy/zzzzzz/.
+       Once the FIG reaches production we can disable this check. */
+    onlyOn( Boolean( fig.url() ), () => {
```

Comment:
I know this was done to variable-ize the FIG URL, but I love the added bonus of how much simpler the skip test logic has become by using an environment variable.

Reply:
Thank you btw @niqjohnson for fixing these tests in the first place and preventing them from breaking our CI when the FIG isn't available.